ratex/csrc/aten_raf_type.cpp

```diff
@@ -1089,8 +1089,18 @@ at::Tensor LazyNativeFunctions::embedding(const at::Tensor& weight, const at::Te
                                           int64_t padding_idx, bool scale_grad_by_freq,
                                           bool sparse) {
   LTC_FN_COUNTER("raf::");
-  // We route embedding to native so that it will be decomposed to supported Lazy operations.
-  return at::native::embedding(weight, indices, padding_idx, scale_grad_by_freq, sparse);
+  LazyTensor weight_tensor;
```

Comment:
Can we add assert if any of padding_idx, scale_grad_by_freq, and sparse parameters are by default

Reply:
Fallback is fine. We can add a warning or a verbose log here.